### PR TITLE
v1.0.1

### PR DESCRIPTION
**Pull request by broodingcoin**
Updated website skeleton and added Bulma CSS

### CODE REVIEW

File: navBar.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+            <a href="html/category1.html">category1</a>
+            <a href="html/livability.html">Livability</a>
+            <a href="html/events.html">Events</a>
+          </div>
+        </div>
+        <li><a href="about.html"></a>About</li>
+      </ul>
+    </nav> -->
+    <!-- Navbar -->
+    <nav class="navbar is-spaced has-shadow is-mobile">
```
Comment (frcampanelli) [inline navBar.html:36]: According to the docs at https://bulma.io/documentation/components/dropdown/
it looks like you need to contain the dropdown in a 'dropdown-trigger' with 'is-hoverable' so it knows to keep the downdown active while the mouse is hovering over it

**Review by frcampanelli — COMMENTED** [review]
NavBar not hovering correctly

File: navBar.html [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      <div class="navbar-brand">
+        <a class="navbar-item title title-padding">CityView+</a>
+      </div>
+      <div class="navbar-menu">
+        <div class="navbar-end">
+          <div class="navbar-item has-dropdown is-hoverable">
```
Comment (frcampanelli) [inline navBar.html:42]: Add dropdown-trigger here

File: navBar.html (hunk 1 above)
Comment (macelott) [reply navBar.html:36]: We can do the dropdown in one of two ways. I am doing it the way it is documented at https://bulma.io/documentation/components/navbar/ which is specific to the navbar

**Comment by macelott** [comment]
Nothing out of place here

**Review by macelott — COMMENTED** [review]
Nothing out of place

**Pull request merged**